webservice/s3_file_handler.py

```diff
@@ -0,0 +1,91 @@
+#! /usr/bin/env python
+
+import boto3
+import botocore.session
+
+
+class S3FileHandler:
+
+    def __init__(self, location):
+        """Expects AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY to be in 
```

Comment:
*How will this interact with how we currently pass in environment variables to the docker container?* 
My understanding is that the `boto3` SDK takes care of that, including throwing an exception if a user hasn't run `aws config` on the command line (or doesn't hold an AWS account), which results in the creation of the credentials file under `~/.aws/config`.

*Do the environment variables need to have certain names, and it will just work?*
*Can you elaborate on the expected environment variable names?*
*For the config dictionary, where does it look for that? Is it a global?*
 (see answer above)

Reply:
This needs to work with CGP deployment. Please see https://github.com/DataBiosphere/cgp-deployment/blob/develop/boardwalk/prod.yml#L48. We will need to restore that in this branch, including the prompts in install_bootstrap and the template file, and read the values from there.